source/configuration_manual/quota/quota_configuration.rst

```diff
@@ -0,0 +1,506 @@
+parameters (if any):
+
+.. code-block:: none
+
+  quota = <backend>[:<quota root name>[:<backend args>]]
+
+The quota root name is just an arbitrary string that is sent to IMAP clients,
+which in turn may show it to the user. The name has no meaning. By default, an
+empty string is used, but you may want to change that since some clients
+(Apple Mail) break and don't show quota at all then.
+
+You can define multiple quota roots by appending an increasing number:
+
+.. code-block:: none
+
+  plugin {
+    quota = maildir:User quota
+    quota2 = fs:Disk quota
+    #quota3 = ...
+  }
+
+Globally available arguments for ``<backend args>`` parameter:
+
+=============== ================================================================
+Name            Description
+=============== ================================================================
+``noenforcing`` Don't try to enforce quotas by calculating if saving would get
+                user over quota. Only handle write failures.
+=============== ================================================================
+
+If you want to specify multiple backend arguments, separate them with ':'
+(e.g. ``noenforcing:foo:bar``).
+
+.. _`IMAP Quota specifications (RFC 2087)`: https://tools.ietf.org/html/rfc2087
+
+Quota Rules
+^^^^^^^^^^^
+
+Quota rules configure the actual quota limits. The syntax is:
+
+.. code-block:: none
+
+  quota_rule = <mailbox name>:<limit configuration>
+  #quota_rule2 = ...
+  #quota_rule3 = ..etc..
+
+``*`` as the mailbox name configures the default limit, which is applied on
+top of a mailbox-specific limit if found. For example:
+
+.. code-block:: none
+
+  quota_rule = *:storage=1G
+  quota_rule2 = Trash:storage=+100M
+  quota_rule3 = SPAM:ignore
+
+This means that the user has 1GB quota, but when saving messages to Trash
+mailbox it's possible to use up to 1.1GB of quota. The quota isn't
+specifically assigned to Trash, so if you had 1GB of mails in Trash you could
+still save 100MB of mails to Trash, but nothing to other mailboxes. The idea
+of this is mostly to allow the clients' move-to-Trash feature work while user
+is deleting messages to get under quota.  Additionally, any messages in the
+SPAM folder are ignored per the ``ignore`` directive and would not count
+against the quota.
+
+``?`` as the mailbox name works almost like ``*``. The difference is that
+``?`` is used only if quota backend doesn't override the limit. For example
+with `Maildir++ quota` if ``maildirsize`` file exists, the limits are taken
+from it, but if it doesn't exist the ``?`` limits are used.
+
+``*`` and ``?`` wildcards can be used as a generic wildcard in mailbox
+names, so for example ``box*`` matches ``boxes``. As shown in the above
+example, the first quota rule is named ``quota_rule`` while the following
+rules have an increasing digit in them. You can have as many quota rules as
+you want.
+
+.. _`Maildir++ quota`: https://www.courier-mta.org/imap/README.maildirquota.html
```

Comment:
This looks .. weird. Does it work? It's a label but points to an external URL. I see it's used elsewhere here as well.. In any case, seems not-so-nice label with `+` and space. Maybe just "maildir_quota"?

Reply:
Works fine.  See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links
I prefer since it makes it easier to read text version of docs (inline links are irrelevant there and make it hard to read the content).
There's nothing wrong with syntax (you can have whatever you want inside of \` except \`).
That being said ... there's an error on line 87 where the link reference is not suffixed with an _, which was preventing the link from appearing.  Fixed.
It is up to you or Aki whether you want to accept this from a style perspective.  But it is valid from a syntax perspective.

Reply:
The problem with your suggestion is that nothing else uses this. So if we accept this style, then we get even more differences of style in documentation compounding the existing problem of syntax differences and notations. I do appreciate your arguments for readability.

Reply:
Now that I tested this, it doesn't look any different in the html output. So it's just syntax difference in the rst file. So I don't really have an objection to it.

Reply:
For sake of moving this merge along, I have converted to `maildir_quota` anonymous link reference name.